test/test_environment.py

```diff
@@ -121,7 +120,7 @@ def test_overridden_pip_constraint(tmp_path, build_frontend):
         setup_py_add=textwrap.dedent(
             """
             import pytz
-            assert pytz.__version__ == "2022.4"
+            assert pytz.__version__ == "2022.4", f"{pytz.__version__!r} != '2022.4'"
```

Comment:
Did this just work already?

Reply:
the a change in pyodide.py in commit https://github.com/pypa/cibuildwheel/pull/1456/commits/4ac060d60361a36b2f33cbff3d16b87ee8357558 fixed this test. I just added some debug string/help if this test fails again.

Reply:
Okay I really appreciate you helping because the last time I tried to fix these I just got so confused.